packages/bbob-parser/src/parse.ts

```diff
@@ -186,6 +186,7 @@ function parse(input: string, opts: ParseOptions = {}) {
     flushTagNodes();
 
     const tagNode = TagNode.create(token.getValue(), {}, []);
+    tagNode.setStart(token.getStart(), token.getEnd());
```

Comment:
I think we can move this to `TagNode.create` as last param `{ start: token.getStart(), end: token.getEnd() }`

Reply:
Added as optional param for static `TagNode.create()`. I've left the constructor alone since it feels better to keep it as required params only.